pepdbagent/modules/namespace.py

```diff
@@ -204,3 +212,54 @@ def info(self, limit: int = DEFAULT_LIMIT_INFO) -> ListOfNamespaceInfo:
             limit=limit,
             results=list_of_results,
         )
+
+    def stats(self, namespace: str = None, monthly: bool = False) -> NamespaceStats:
+        """
+        Get statistics for project in the namespace or for all projects in the database.
+
+        :param namespace: namespace name [Default: None (all projects)]
+        :param monthly: if True, get statistics for the last 3 years monthly, else for the last 3 months daily.
```

Comment:
Just curious, why these default values? 3 years monthly vs 3 months daily.
Why not just the last year? or the last 6 months if monthly?

Reply:
idk, I just decided to do it. We can change it in the future. This functionality will be disabled on pephub anyway.
About 3 month, Nathan suggested it